# My Jupyter Notebook on IBM Watson Studio

**Allan Paolo P. Pomperada** \
Well Drilling Engineer

*I am interested in Data Science because I am a naturally curious person. With Data Science, I can utilize my curiousity in providing insights to complex data to present it in a simple manner. I also love solving problems which is the essence of Data Science*

### I would be doing some simple BMI calculation with two sets of list.


In [3]:
height = [1.63, 1.59, 1.60]
weight = [56, 63, 59]


import numpy as np

np_height = np.array(height)
np_weight = np.array(weight)

# BMI = weight in kg / (height in m ^2)

bmi = np_weight / (np_height ** 2)


print(bmi)

[21.07719523 24.91990032 23.046875  ]


From the given data

height|weight
---|---
1.63|56
1.59|63
1.60|59

> The result gives us the following BMI respectively:
- 21.07719523
- 24.91990032
- 223.046875

<img src="https://img.freepik.com/premium-vector/indikator-bmi-white-background-chart-concept-vector-icon_100456-6267.jpg?w=360" alt="BMI logo" style="height: 100px; width:100px;"/>


